### ISSUE TITLE
Add option to send SMS to admin if Ticket Priority is High

### ISSUE THREAD

**Issue by starnetwork** [AktuelSistem/WHMCS-SmsModule#88]
Hi,
Please add ticket priority hook support and SMS to admin

**Comment by shibby** [comment]
With current structure we cannot do this request.

**Comment by starnetwork** [comment]
why, it's not possible to add it as additional hook?
whmcs support it: http://docs.whmcs.com/Hooks:TicketPriorityChange

**Comment by shibby** [comment]
Yes this is a way :) I'm labeling it as enhancement. Maybe with next version, we can add this. And, i don't know when next version will released :/
